instrumentation/httpurlconnection/library/src/main/java/io/opentelemetry/instrumentation/library/httpurlconnection/HttpUrlInstrumentationConfig.java

```diff
@@ -5,15 +5,16 @@
 
 package io.opentelemetry.instrumentation.library.httpurlconnection;
 
-import io.opentelemetry.instrumentation.api.incubator.semconv.net.PeerServiceResolver;
-import io.opentelemetry.instrumentation.api.internal.HttpConstants;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
+import io.opentelemetry.instrumentation.api.incubator.semconv.net.PeerServiceResolver;
```

Comment:
Question - Any idea why spotlessApply felt the need to move the imports? They were not touched in this change.